app/components/links/LinkText.tsx

```diff
@@ -8,7 +8,10 @@ interface LinkTextProps {
 
 const LinkText: FC<LinkTextProps> = ({ url, text }) => {
   return (
-    <Link href={url} className="font-normal text-sm leading-5 text-skyBlue">
+    <Link
+      href={url}
+      className="font-normal text-sm leading-5 text-skyBlue hover:underline"
+    >
```

Comment:
Next.js automatically applies href, but explicitly passing it as a prop improves screen reader support in older versions. Read [this](https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag).

```suggestion
      href={url}
      passHref
      className="font-normal text-sm leading-5 text-skyBlue hover:underline"
    >
```

Reply:
using it

Reply:
using it